library/Opus/Doi/include/datacite-contributorType-v4.xsd

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ich werde mir irgendwann überlegen, wie ich diese Dateien usw. in ein separates Package auslagern kann, damit ich bei Datacite-Änderungen nicht gezwungen bin das Framework anzufassen.

Reply:
Ja, das ist wahrscheinlich sinnvoll. Die Schema-Definitionen werden scheinbar im Jahresintervall aktualisiert: https://schema.datacite.org/

Kann man dafür nicht auch ein Composer Projekt anlegen, das nur die XSDs von DataCite enthält?

Reply:
Ja, im Prinzip geht das. Ich würde auch noch den DataCite-Code hinzufügen, zumindest den allgemeinen, insbesondere Funktionen, um die XSD-Dateien zu finden.